src/Components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import "./navbar.css";
+import { AiFillCloseCircle } from "react-icons/ai";
+import { TbGridDots } from "react-icons/tb";
+import kp1 from "../../Assets/kp1.png";
+
+interface NavbarProps {}
+
+const Navbar: React.FC<NavbarProps> = () => {
+  const [active, setActive] = useState(false);
+
+  // Function to toggle Navbar
+  const toggleNav = () => {
+    setActive((prevState) => !prevState);
+  };
+
+  return (
+    <section className={`navBarSection ${active ? "activeNavbar" : ""}`}>
```

Comment:
const Logo = () => (
  <div className="logoDiv">
    <a href="#" className="logo flex">
      <img src={kp1} alt="KP" />
      <h1 className="icon">Kalpavriksha</h1>
    </a>
  </div>
);

const NavItem = ({ text }) => (
  <li className="navItem">
    <a href="#" className="navLink">
      {text}
    </a>
  </li>
);

const LoginButton = () => (
  <button className="btn">
    <a href="#">Log-In</a>
  </button>
);
/........
const Navbar: React.FC = () => {
  const [active, setActive] = useState(false);

  const toggleNav = () => {
    setActive((prevState) => !prevState);
  };

  return (
    <section className={`navBarSection ${active ? "activeNavbar" : ""}`}>
      <header className="header flex">
        <Logo />
        <div className={`navBar ${active ? "activeNavbar" : ""}`}>
          <ul className="navLists flex">
            <NavItem text="Home" />
            <NavItem text="Courses" />
            <NavItem text="Coding" />
            <NavItem text="Certification" />
            <NavItem text="Pricing" />
            <LoginButton />
          </ul>
          <div onClick={toggleNav} className="closeNavbar">
            <AiFillCloseCircle className="icon" />
          </div>
        </div>
        <div onClick={toggleNav} className="toggleNavbar">
          <TbGridDots className="icon" />
        </div>
      </header>
    </section>
  );
};

export default Navbar;


See if we can reuse the componets